src/components/common/header/header.scss

```diff
@@ -163,7 +163,26 @@ header {
     }
 
     
-  }
+  } 
+
+  
+}  
+
+.popupdisplayHIDE{
+  display: none
```

Comment:
we already have a class for hiding in header.scss ".hide"

Reply:
Will refactor to use .hide